homeassistant/components/scene/homeassistant.py

```diff
@@ -92,7 +70,7 @@ def _process_config(scene_config):
     return SceneConfig(name, states)
 
 
-class Scene(Entity):
+class HomeAssistantScene(Scene):
```

Comment:
For this component, get rid of `should_poll`, `state`, `entity_ids` and rename `state_attributes` to `device_state_attributes` (inline `entity_ids` property in dict)